docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
+    section, where we provide links to a variety of books and papers.
+    In our :ref:`user guide <user_guide>` we also have links to the papers describing
+    Fairlearn's mitigation algorithms.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* uncover these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    If a model is unfair due to biased data (note that this is not the only reason a model
```

Comment:
Again the first step in answering the question is to dissect the notion of fairness. The model can be "unfair" in different ways. I am not sure if the notion of fairness here is related to the harms of allocation or harms of quality of service. The answers will be a bit different. Also, we need to clarify what is meant by "biased data"--this can mean different things. Does this refer to the fact that different groups might not be sufficiently well represented (so data is skewed towards certain groups)? Or does this mean that the labels are generated by some process that needs to be accounted for (otherwise a statistical bias ensues).

Reply:
Updated with an example